R/get_entity.R

```diff
@@ -1,165 +1,32 @@
-        bulk_get_response_content <- NA
-        bulk_get_response_check <- 0
-      }
-    } else {
-      bulk_get_response_content <- NA
-      bulk_get_response_check <- 0
-    }
-    if (single_get_response_check == 1 | bulk_get_response_check == 1) { 
-      if (single_get_response_check == 1 & bulk_get_response_check == 0) {
-        response_list <- list(entity[1], single_get_response_content, single_get_response_content[["name"]])
-        names(response_list)[1] <- "Entity Schema"
-        names(response_list)[2] <- "Single-Get API Endpoint Response Content"
-        names(response_list)[3] <- "Entity Name"
-      }
-      if ((single_get_response_check == 0 & bulk_get_response_check == 1) | 
-          (single_get_response_check == 1 & bulk_get_response_check == 1)) {
-        response_list <- list(entity[1], bulk_get_response_content, bulk_get_response_content[[1]][[1]][["name"]])
-        names(response_list)[1] <- "Entity Schema"
-        names(response_list)[2] <- "Bulk-Get API Endpoint Response Content"
-        names(response_list)[3] <- "Entity Name"
-      }
-    } else {
-      response_list <- list(entity[1], NA, NA)
-      names(response_list)[1] <- "Entity Schema"
-      names(response_list)[2] <- "Single-Get API Endpoint/Bulk-Get API Endpoints Response Content"
-      names(response_list)[3] <- "Entity Name"
-    }
-    return(response_list)
-  }
-  for (i in 1:length(inferred_types)) {
-    if (is.na(inferred_types[[i]][1]) | is.na(inferred_types[[i]][2]) | is.na(inferred_types[[i]][3])) {
-      if (is.na(inferred_types[[i]][1])) {
-        message1 <- paste0("Entity schema for ", as.character(names(inferred_types)[[i]]), " is unknown and labeled as NA. ")
-      } else {
-        message1 <- ""
-      }
-      if (is.na(inferred_types[[i]][2])) {
-        message2 <- paste0("Single-Get API endpoint for ", as.character(names(inferred_types)[[i]]), " is unknown and labeled as NA. ")
-      } else {
-        message2 <- ""
-      }
-      if (is.na(inferred_types[[i]][3])) {
-        message3 <- paste0("BulkGet API endpoints for ", as.character(names(inferred_types)[[i]]), " are unknown and labeled as NA.")
-      } else {
-        message3 <- ""
-      }
-      warning(paste0(message1, message2, message3))
-    }
-    if (is.na(inferred_types[[i]][1]) & is.na(inferred_types[[i]][2]) & is.na(inferred_types[[i]][3])) {
-      entities[[i]] <- list(NA,NA,NA)
-      names(entities[[i]])[1] <- "Entity Schema"
-      names(entities[[i]])[2] <- "Single-Get API Endpoint/Bulk-Get API Endpoints Response Content"
-      names(entities[[i]])[3] <- "Entity Name"
-      names(entities)[i] <- names(inferred_types)[i]
-    } else {
-      entities[[i]] <- .get_entity_from_id(entity=inferred_types[[i]], api_key=benchling_api_key)
-      names(entities)[i] <- names(inferred_types)[i]
-    }
-  }
-  return(entities) # list of lists
+get_entity <-  function(entity_id, entity_list=NULL, benchling_api_key=Sys.getenv("BENCHLING_API_KEY")) {
```

Comment:
Please change `benchling_api_key` to `api_key` here and remove the `entity_list` argument for now.